### ISSUE TITLE
Possible inconsistency in school escorting

### ISSUE THREAD

**Issue by jpn--** [ActivitySim/activitysim#757]
**Describe the bug**
The `PERSON_WEIGHT` setting if provided in `school_escorting.yaml` is incorrectly applied as both the person weight and the gender weight.

**Expected behavior**
Gender weight should be its own setting.

**Details**
In the code [here](https://github.com/ActivitySim/activitysim/blob/a8e755f96d0e32633a6d3657c4878e3b6a37e59a/activitysim/abm/models/school_escorting.py#L49) we have:

```
    persontype_weight = model_settings.get("PERSON_WEIGHT", 100)
    gender_weight = model_settings.get("PERSON_WEIGHT", 10)
    age_weight = model_settings.get("AGE_WEIGHT", 1)
```

That middle one is clearly meant to be “GENDER_WEIGHT”.   I’m going to fix it in our configs task PR.  

In the test `school_escorting.yaml`, none of these weights are set in the settings file, so we’re getting the default values of 100/10/1.  Therefore when I fix it nothing in our test or consortium maintained examples will appear to change or break because of the fix.  But if there’s somebody’s implementation out there that’s using “PERSON_WEIGHT” or “GENDER_WEIGHT”, the fix may change results.  

cc: @dhensle @i-am-sijia

**Comment by dhensle** [comment]
Hey @jpn--, thanks for pointing this out.  This was already fixed in the [latest BayDAG improvements](https://github.com/SANDAG/activitysim/pull/58/files#diff-1d67ac6f064a19a419b67fc8c36b56a9a1a2f18cc4dc2ed0588876fc9b1ea713), but extracting to its own issue here is fine with me.

**Comment by dhensle** [comment]
Fixed with v1.3 release.
